nix/haskell.nix

```diff
@@ -60,7 +60,6 @@ let
       src = ../.;
       name = "cardano-db-sync";
     };
-    ghc = buildPackages.haskell-nix.compiler.${compiler};
     modules = [
```

Comment:
I think you probably want to add `inherit compiler-nix-name;` here

Reply:
Done :-)